modules/cluster-logging-collector-log-forward-configure.adoc

```diff
@@ -17,60 +17,56 @@ To configure log forwarding using the API:
 +
 [source,yaml]
 ----
-apiVersion: "logging.openshift.io/v1alpha1"
-kind: "LogForwarding"
+apiVersion: "logging.openshift.io/v1"
+kind: "ClusterLogForwarder"
 metadata:
   name: instance <1>
   namespace: openshift-logging <2>
 spec:
-  disableDefaultForwarding: true <3>
-  outputs: <4>
+  outputs: <3>
    - name: elasticsearch
      type: "elasticsearch"
-     endpoint: elasticsearch.openshift-logging.svc:9200
+     url: https://elasticsearch.openshift-logging.svc:9200
      secret:
         name: fluentd
```

Comment:
@mburke5678 could you please remove this output? In clusterlogforwarder, we have a default output named as `default`, if user want to set forward logs to default ES, just need to add `- default` to the `outputRefs`, no need to  define an output. You can find the details in https://issues.redhat.com/browse/LOG-706.

Reply:
Removed this file.